### ISSUE TITLE
Default line height broken

### ISSUE THREAD

**Issue by bluebrown** [tailwindcss/tailwindcss#1626]
Hi, I am trying to set the default line height like shown [in the docs](https://tailwindcss.com/docs/font-size/#providing-a-default-line-height)

## tailwind.config.js

```javascript
fontSize: {
    test: ['16px', '24px'],
 },
```

## result

```css
.text-test {
    font-size: 16px;
    font-size: 24px;
}
```

![image](https://user-images.githubusercontent.com/39703898/80307213-8fd7a880-87bf-11ea-8ad1-1aab8bc16e3e.png)

**Comment by adamwathan** [comment]
What version of Tailwind are you using?

**Comment by bluebrown** [comment]
Oh sorry Im using > tailwindcss": "^1.2.0"

**Comment by adamwathan** [comment]
That feature is only available in 1.3.0+, so make sure you upgrade and then it should work 👍🏻 Forgot to mention that in the docs, my mistake!

**Comment by bluebrown** [comment]
Ah, that was only released a couple of days ago It seems. Yes, I upgraded and it works as expected.
